packages/indexer-common/src/allocations/types.ts

```diff
@@ -1,28 +0,0 @@
-import { Address } from '@graphprotocol/common-ts'
-import { BigNumber } from 'ethers'
-import { SubgraphDeployment } from '@graphprotocol/indexer-common'
-
-export interface Allocation {
-  id: Address
-  status: AllocationStatus
-  subgraphDeployment: SubgraphDeployment
-  indexer: Address
-  allocatedTokens: BigNumber
-  createdAtEpoch: number
-  createdAtBlockHash: string
-  closedAtEpoch: number
-  closedAtEpochStartBlockHash: string | undefined
-  previousEpochStartBlockHash: string | undefined
-  closedAtBlockHash: string
-  poi: string | undefined
-  queryFeeRebates: BigNumber | undefined
-  queryFeesCollected: BigNumber | undefined
-}
-
-export enum AllocationStatus {
-  NULL = 'Null',
-  ACTIVE = 'Active',
-  CLOSED = 'Closed',
-  FINALIZED = 'Finalized',
-  CLAIMED = 'Claimed',
-}
```

Comment:
Why was this file removed? I think we should have both `types.ts` and `monitor.ts`

Reply:
@carlosvdr yup I made a mistake in the review, I was somehow convinced that this eventual was implemented twice. We need the file back :grimacing: